src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/tests/test_webapp_commands.py

```diff
@@ -76,6 +76,65 @@ def body(self):
             ])
         self.cmd('webapp delete -g {} -n {}'.format(self.resource_group, webapp_name))
         #test empty service plan should be automatically deleted.
-        result = self.cmd('webapp plan list -g {}'.format(self.resource_group), checks=[
+        result = self.cmd('app-service-plan list -g {}'.format(self.resource_group), checks=[
             JMESPathCheck('length(@)', 0)
             ])
+
+class WebappConfigureTest(ResourceGroupVCRTestBase):
+
+    def __init__(self, test_method):
+        super(WebappConfigureTest, self).__init__(__file__, test_method)
+        self.resource_group = 'azurecli-webapp-config'
+        self.webapp_name = 'webapp-config-test'
```

Comment:
you can right. I removed the `set_up` method by mistake before I submitted PR. Adding it back.